src/main/java/com/example/javaOjt/beans/requests/AuthorRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.example.javaOjt.beans.requests;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AuthorRequest {
+
+  @NotBlank
+  String name;
```

Comment:
追加で `name` を private にしましょう！
```suggestion
  private String name;
```